run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.fullchem

```diff
@@ -3863,7 +4061,7 @@ ${RUNDIR_TES_CLIM_N2O}
 #==============================================================================
 # --- Annual scale factors ---
 #==============================================================================
-(((HTAP.or.XIAO_C3H8
+(((HTAPv3.or.HTAPv3_SHIP.or.XIAO_C3H8
```

Comment:
See my comment for this section in HEMCO_Config.rc.aerosol. I think we can remove the HTAPv3 logicals here and also remove unused scale factor entries.

Reply:
@yantosca Could you also remove the obsolete scale factors from this section? That is, only scale IDs 6 and 7 should remain since those are the only annual scale factors used by XIAO_C3H8.